### ISSUE TITLE
style variable fallthrough seems broken. 

### ISSUE THREAD

**Issue by newlifehaonan** [icehaunter/vue3-datepicker#27]
the styling playground seems broken, example doesn't reflect changes when variable value changed
![Screen Shot 2021-02-19 at 3 40 25 PM](https://user-images.githubusercontent.com/32348307/108564879-6e552980-72c9-11eb-9d14-db81906fbd0a.png)
also tried to specified variable value in my project like this, doesn't seems to work.
```
<Datepicker v-model="picked" style="--vdp-divider-color:#000000"/>
```

**Comment by icehaunter** [comment]
Strange, I'll look into that. It worked before :sweat_smile:

**Comment by Talbourdet** [comment]
I have the same issue @newlifehaonan Did you find a solution

**Comment by reazulackosoft** [comment]
I was also facing same problem. I have overwritten it by the following way.
```
<style>
.v3dp__datepicker {
  --elem-hover-bg-color: var(--vdp-hover-bg-color, #62d1f3);
  --elem-selected-bg-color: var(--vdp-selected-bg-color, #4680ff);
}
</style>
<template>
    <datepicker v-model="picked"/>
</template>
```

**Comment by icehaunter** [comment]
Thanks for the report, sorry for the delay, will be fixed in the next release

**Comment by icehaunter** [comment]
Yeah, published under 0.2.5. You can check the playground in the docs to verify that everything is fixed :D
